### ISSUE TITLE
Can't add log source from file.

### ISSUE THREAD

**Issue by Gimi** [splunk/splunk-connect-for-kubernetes#14]
Behavior:
* In `values.yaml`, add some log source from file like
```yaml
logs:
  audit-logs:
    from:
      file:
        path: /var/log/audit/*.log
```
* Install the logging chart with that `values.yaml`

Expect:
Data in files match the pattern `/var/log/audit/*.log` should be ingested.

Actual:
Error raised:
```
2018-06-14 19:36:48 +0000 [error]: config error file="/fluentd/etc/fluent.conf" error_class=Fluent::ConfigError error="Could not parse jq filter: .record.sourcetype = (.tag | ltrimstr(\"tail.file.\") | .record, error: Could not parse jq filter: .record.sourcetype = (.tag | ltrimstr(\"tail.file.\") | .record, error: jq: error: syntax error, unexpected $end (Unix shell quoting issues?) at <top-level>, line 1:\n.record.sourcetype = (.tag | ltrimstr(\"tail.file.\") | .record                                                      \njq: 1 compile error\n"
```

**Comment by Gimi** [comment]
This is caused by a syntax error in the file log jq filter.

**Comment by Gimi** [comment]
Fixed in 68ed2fdfdbdc91a7df4a651ae73de51e6e1eb336
